main.go

```diff
@@ -235,32 +261,33 @@ func downloadSourcePaths(sourcePaths []string, destPath string, githubRepo GitHu
 	return nil
 }
 
-// Download the specified binary files that were uploaded as release assets to the specified GitHub release
-func downloadReleaseAssets(releaseAssets []string, destPath string, githubRepo GitHubRepo, latestTag string) error {
-	if len(releaseAssets) == 0 {
-		return nil
+// Download the specified binary file that was uploaded as a release asset to the specified GitHub release.
+// Returns the path where the release asset was downloaded.
+func downloadReleaseAsset(assetName string, destPath string, githubRepo GitHubRepo, tag string) (string, error) {
+	var assetPath string
+
+	if assetName == "" {
```

Comment:
Should this be an error?

Reply:
No. This code got updated when I changed `assetNames` from a `[]string` to a `string`. That being said, I think the error handling could generally be improved here, as discussed above.